components/create.js

```diff
@@ -1,31 +1,25 @@
 import React from "react";
-import DefineMap from "can-define/map/";
-import Component from "react-view-model/component";
 import Todo from "../models/todo";
+import { connect, ObserveObject } from "ylem";
 
-export const ViewModel = DefineMap.extend("TodoCreateVM", {
-	todo: { Value: Todo },
-});
-
-export default class Create extends Component {
-	createTodo(e) {
+class Store extends ObserveObject {
```

Comment:
How do people feel about calling these `Store` everywhere?  This will probably be very annoying to debug.  Can we call these `CreateTodoVM` or `TodoCreate`?

Reply:
Since the file and component are `Create`,  I would call this `CreateStore` or `CreateViewModel`/`CreateVM`. The ylem docs are specifically calling things store over view-model, as ylem can be used any number of ways, but the specific example can certainly use VM if that is appropriate.

However you decide to resolve this, I think there are a couple of applicable places.

Reply:
I like `Store` over `ViewModel`, especially in react, a lot of devs will have some familiarity with Redux Stores. `CreateStore` seems like the way to go